src/Nova/Location.php

```diff
@@ -97,6 +99,26 @@ protected function gmbFields()
         ];
     }
 
+    protected function hoursFields()
```

Comment:
<img width="1241" alt="Screen Shot 2021-04-12 at 10 05 48 PM" src="https://user-images.githubusercontent.com/24723492/114490891-3dafb300-9bdb-11eb-94a0-b5a0a72d9a91.png">

Reply:
That looks good! We should probably update the gmb_hours factory so it provides more accurate string data.

Reply:
Since they are accessor fields, we could just do one for every day of the week. If the fields are NULL for the day, then have it returned "Closed" and if it is open then return "10:00 AM to 9:00 PM"

Reply:
https://github.com/tipoff/locations/pull/110/commits/8ba98581eecc584dc8896057c99324d725fb7d1b

<img width="1136" alt="Screen Shot 2021-04-13 at 8 19 15 AM" src="https://user-images.githubusercontent.com/24723492/114559042-f0f6c700-9c30-11eb-8aaa-53e9c0a1708e.png">

Reply:
That's awesome! 🚀

Reply:
Minor change to those accessors is to remove leading zeroes and add a space between the time and AM or PM. Also convert to 12-hour time format instead of military time.

Reply:
https://github.com/tipoff/locations/pull/110/commits/b6fac7785a52667608fb4106c491fa0e571d9cee